datalake/backend/endpoints/utils.py

```diff
@@ -13,3 +14,10 @@ def success_response(code, body):
     """Return success response content as string."""
 
     return {"statusCode": code, "body": body}
+
+
+class ResourceName(Enum):
+    DATASETS_TABLE_NAME = "datasets"
```

Comment:
Could we add `DEPLOY_ENV` to the start of these names *here?* We'll need to move the constant from app.py here to avoid a circular import.